app/javascript/scimo/styles/Diffusion.scss

```diff
@@ -0,0 +1,192 @@
+/* ==========================================================================
```

Comment:
I think this css file can be deleted actually since I don't think any selectors are being used